lib/parse/index.js

```diff
@@ -56,18 +56,21 @@ module.exports = function(css, options){
    * Error `msg`.
    */
 
-  function error(msg) {
-    if (options.silent === true) {
-      return false;
-    }
+  var errorsList = [];
 
+  function error(msg) {
     var err = new Error(options.source + ':' + lineno + ':' + column + ': ' + msg);
     err.reason = msg;
     err.filename = options.source;
     err.line = lineno;
     err.column = column;
     err.source = css;
-    throw err;
+
+    if (options.silent === true) {
```

Comment:
The rest of the code base just check for truthiness of boolean options. I’d suggest removing ` === true` for consistency.